styles.css

```diff
@@ -16,16 +16,13 @@ h2 {
 h3 {
 }
 
-nav {
+nav ul {
   display: flex;
   flex-direction: row;
-  justify-content: space-between;
-  width: 20em;
   float: right;
-}
-
-nav a {
-  padding: 10px;
+  justify-content: space-between;
+  margin: auto;
+  width: 22em;
 }
```

Comment:
Some of this stuff in the `nav` didn't work since there was also a `ul` so I moved the styling to the `ul`.

Reply:
Ok